### ISSUE TITLE
text confusion in Persian/Arabic text that has diacritics!

### ISSUE THREAD

**Issue by 9px** [peter-iakovlev/Telegram#191]
i use last version of telegram (4.5) on iPad mini 4 with iOS 11.

i see text confusion in Persian/Arabic text that has diacritics!

see these two image for example...

![1](https://user-images.githubusercontent.com/1096849/33230661-dbea3c68-d1fc-11e7-8901-5010e30e401c.png)

![2](https://user-images.githubusercontent.com/1096849/33230663-e209fca0-d1fc-11e7-9b57-4e2f0a2bf11b.png)

example text:

به دلالت خداى متعال در كلامش همه ی مخلوقات و موجودات، داراى عقل و فهم و شعورند. و يك شاهد بر اين امر تسبيح آن ها است؛ براى نمونه:
تُسَبِّحُ لَهُ السَّماواتُ السَّبْعُ وَ الْأَرْضُ وَ مَنْ فيهِنَّ وَ إِنْ مِنْ شَيْ‏ءٍ إِلاَّ يُسَبِّحُ بِحَمْدِهِ وَ لكِنْ لا تَفْقَهُونَ تَسْبيحَهُم‏...(اسراء: ۴۴) 
و از طرف ديگر وسعت جغرافيائى خلقت، و انتهاى بى انتهاى آن كه فرمود:" إِنَّا خُلِقْنَا وَ إِيَّاكُمْ لِلْبَقَاءِ لَا لِلْفَنَاء"  (أمالي شیخ طوسي: ۲۱۶، از امیرالمومنین علیه السّلام) عظمتى را رقم مى زند كه به جز حيرت سرسام آور نصيب طالب تصوّر آن نمى كند

**Comment by ehsanshb** [comment]
Yes. I had same issue in ios as well. I guessed that ios was interrupted Arabic text after upgrade to ver. 11. But I copied same Arabic text with diacritics from Telegram to WhatsApp, Apple Notes, Viber, etc and no issues had been shown. So , I believe Telegram has problem with same Arabic texts.

**Comment by vsg24** [comment]
This is horrible. Needs a fix very soon.
